pkg/pgprem/adapter.go

```diff
@@ -212,7 +206,7 @@ func (adapter *DefaultPostgreSQLAdapter) ApplyConfig(proposedConfig *agent.Propo
 
 			sudoCmd := exec.Command("sudo", "systemctl", "restart", adapter.pgConfig.ServiceName)
 			if sudoErr := sudoCmd.Run(); sudoErr != nil {
-			  return fmt.Errorf("failed to restart PostgreSQL service with sudo: %w", sudoErr)
+				return fmt.Errorf("failed to restart PostgreSQL service with sudo: %w", sudoErr)
```

Comment:
@mbocevski check your editor/`coc` setup for `go`. Not sure `go fmt` has a config file as thats kind of the point of it.